src/driver/sqljs/SqljsQueryRunner.ts

```diff
@@ -31,6 +31,15 @@ export class SqljsQueryRunner extends AbstractSqliteQueryRunner {
     // -------------------------------------------------------------------------
     // Public methods
     // -------------------------------------------------------------------------
+    
+    /**
+     * Commits transaction.
+     * Error will be thrown if transaction was not started.
+     */
+    async commitTransaction(): Promise<void> {
+        await super.commitTransaction();
+        this.driver.autoSave();
```

Comment:
did you miss comment where I told that autoSave most probably should be a promise?

Reply:
No I didn't.
But I added back then: What would be the difference? I don't know if it would be a good idea to wait for the save operation to complete since it will slow down the driver.

Reply:
it will be correct to wait a save operation to complete, otherwise users may face unexpected behaviour

Reply:
Plus if there will be an error they will see it, it won't go to the air

Reply:
I changed that, to wait for save